src/Aspire.Hosting/Utils/PathNormalizer.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.Utils;
+
+internal static class PathNormalizer
+{
+    public static string NormalizePathForCurrentPlatform(string path)
+    {
+        if (string.IsNullOrEmpty(path))
+        {
+            return path;
+        }
+
+        // Fix slashes
+        path = path.Replace('\\', Path.DirectorySeparatorChar).Replace('/', Path.DirectorySeparatorChar);
```

Comment:
I'm thinking it might be better for us to pick `/` as the normalized separator value. I'm considering scenarios whether you might generate a manifest on one platform but use it from another (build pipelines across multiple agents etc).

Reply:
This isn't used for manifest generation. We'll have another overload that does this.